k8s/README.md

```diff
@@ -72,8 +72,7 @@ To deploy to the cluster:
 
```

Comment:
The previous instructions no longer need to mention that the user needs to make a note of the service configuration ID.